cmd/hostpaths/hostpaths.go

```diff
@@ -420,6 +433,32 @@ func mapHostPaths(ctx context.Context, pManager, vManager manager.Manager) {
 	}, time.Second*5)
 }
 
+func filterManagedPods(ctx context.Context, options *context2.VirtualClusterOptions, podList *corev1.PodList, pManager manager.Manager) error {
```

Comment:
could this actually be just creating a fielduelector for the list operation? 
we can select on the namespace with is `metadata.namespace!=default`, this way we'd only be fetching what we need instead of fetching more and filtering afterwards